pkg/handler/repositories.go

```diff
@@ -60,53 +64,128 @@ func listRepositories(c echo.Context) error {
 // @Tags         repositories
 // @Accept       json
 // @Produce      json
-// @Param  body       body    api.CreateRepository true  "request body"
-// @Param  org_id     header  string         	   true  "organization id"
-// @Param  account_id header  string               true  "account number"
```

Comment:
FYI i removed these org_id & account_id header declarations.  They aren't strictly correct, since it has to be within the identity header.  I don't think they need to be here since the user would actually be passing basic auth with their request and the 3scale infrastructure would add in the identity header, but we can see what other apps are doing